neo/Network/P2P/ProtocolHandler.cs

```diff
@@ -94,6 +94,15 @@ private void OnMessage(Message msg)
                 case "consensus":
                     OnInventoryReceived(msg.GetPayload<ConsensusPayload>());
                     break;
+                case "root":
+                    OnInventoryReceived(msg.GetPayload<StateRoot>());
```

Comment:
Where it's send this message?

Reply:
https://github.com/neo-ngd/neo/blob/99061343bf91b34a8264d32b684fc92f2d343ee7/neo/Consensus/ConsensusService.cs#L123

https://github.com/neo-project/neo/blob/24ecde1ea3942540778d7e226ddfc69407751060/neo/Ledger/BlockChain.State.cs#L64